src/styles/default/pretty.jl

```diff
@@ -283,27 +283,26 @@ end
 
 function format_docstring(style::AbstractStyle, state::State, text::AbstractString)
     state_indent = state.indent
-    boundaries = findall(text) do character
-        character != '"'
-    end
+    start_boundary = findfirst(!=('"'), text)
+    end_boundary = findlast(!=('"'), text)
     # first, we need to remove any user indent
     # only some lines will "count" towards increasing the user indent
     # start at a very big guess
     user_indent = typemax(Int)
-    user_indented = text[boundaries[1]:boundaries[end]]
+    user_indented = text[start_boundary:end_boundary]
```

Comment:
`findfirst` and `findlast` return valid indices of `text` (given that there is a non-`"` character, which would be good to check...)